pkgs/shells/zsh/zplug/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "2.4.2";
+  pname = "zplug";
+
+  src = fetchFromGitHub {
+    owner = "zplug";
+    repo = "zplug";
+    rev = version;
+    sha256 = "0hci1pbs3k5icwfyfw5pzcgigbh9vavprxxvakg1xm19n8zb61b3";
+  };
+
+  phases = "installPhase";
+
+  installPhase = ''
```

Comment:
Please include a `runHook preInstall` and `runHook postInstall` as the first and last line of the `installPhase` if you don't mind.

Reply:
Got it!